tests/crafting_test.cpp

```diff
@@ -753,141 +753,111 @@ TEST_CASE( "crafting_skill_gain", "[skill],[crafting],[slow]" )
 {
     SECTION( "lvl 0 -> 1" ) {
         GIVEN( "nominal morale" ) {
-            test_skill_progression( recipe_id( "blanket" ), 175, 0, false );
```

Comment:
Why are all these tests being removed?

Reply:
This is deliberate. Here is the reason:
The purpose of the test called "crafting_skill_gain" is to check how long it takes to gain a skill level via crafting and detect unintentional regression. When proficiencies were added this test started doing double-duty by also checking the impact of proficiencies. 
These tests are too brittle and will break any time there's a change in the proficiency penalty formulas or proficiency xp gain. This causes excessive maintenance burden since you have to update up to 36 expectations. A single such test should suffice for checking the impact of proficiency on crafting time. We should probably set up a new unit unit test dedicated to measuring how long it takes to level up proficiencies by simulating multiple crafting operations. But the more scenarios you simulate the more values need to be updated when you make any change to the formula -- a lot of work for little benefit.